### ISSUE TITLE
[Feature Request] File types

### ISSUE THREAD

**Issue by Amnesia1000** [mgba-emu/mgba#2359]
Add option to incorporate file types to the system (gb, gbc, gba) and their association to the emulator.

**Comment by endrift** [comment]
The installer already has this.

**Comment by Amnesia1000** [comment]
Ahhh ok, that's something I didn't know. I manage with the git zip

**Comment by endrift** [comment]
Yeah I didn't immediately close this because there's no way to do it _without_ the installer yet, and I wasn't sure if that's what you were asking for.

**Comment by Amnesia1000** [comment]
The truth is that it would be nice to have this option inside the emulator, I think it would be more practical since not everyone uses the installer. But as I don't know about programming I don't know if it is something complicated to do or not.

**Comment by endrift** [comment]
I do not know exactly how to implement them (the installer just has an option for them), but I'm sure I can find out. It shouldn't be too bad.
